### ISSUE TITLE
assert2 failes to link on windows due to isatty dependency

### ISSUE THREAD

**Issue by pdylanross** [de-vri-es/assert2-rs#21]
When consuming this library from a windows based development environment, it fails to compile due to the dependency on isatty with the error:

```
 error LNK2019: unresolved external symbol isatty referenced in function _ZN7assert214__assert2_impl5print13stderr_is_tty17hb9c023091d3e02ccE
```

**Comment by pdylanross** [comment]
Reproduced in an [action run](https://github.com/de-vri-es/assert2-rs/actions/runs/562319625)

[Pull request here](https://github.com/de-vri-es/assert2-rs/pull/22)

**Comment by de-vri-es** [comment]
Thanks for reporting and fixing the issue! Your PR is merged and released as 0.3.5 :)
